examples/MvcWeb/TheraLang/Controllers/ProjectTypeController.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using MvcWeb.Services;
+using MvcWeb.TheraLang.Entities;
+
+namespace MvcWeb.TheraLang.Controllers
+{
+    [Route("api/ProjectType")]
```

Comment:
```suggestion
    [Route("api/projectTypes")]
```
but so not forget update link on front as well